news/2017-08-30-qat-gazette-august-2017.md

```diff
@@ -0,0 +1,41 @@
+# QAT Gazette - August 2017
+
+Hello!
```

Comment:
needs an opener paragraph which is short and sweet and introduces the main purpose of the article to the readers. check out https://osu.ppy.sh/home for an example of what these openers should be

Reply:
Added some juicy teasers